### PR TITLE
Views details

### PR DESCRIPTION
**Pull request by BenM4k**
In this update, I added details to my view components, following the wireframe logic. For more details check out [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -10,5 +10,6 @@
         </li>
     <% end %>
     <br/>
+    <br/>
     <%= link_to 'show more', user_posts_path(@user.id) %>
 </ul>
```
Comment (Qoosim) [inline app/views/users/show.html.erb]: - [ ] You're required to display user's 3 most recent posts on the user show  page. Kindly attend to that.

### Area of attention :point_down: 

recent-post

*Hint: * It should look something similar to this :point_down: 
```
  <% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

```

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -10,5 +10,6 @@
         </li>
     <% end %>
     <br/>
+    <br/>
     <%= link_to 'show more', user_posts_path(@user.id) %>
```
Comment (Qoosim) [inline app/views/users/show.html.erb]: - [ ] Kindly change the text to `See all Post` as we have it on the wireframe

see-all-post

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @BenM4k, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Implemented some project requirements. :heavy_check_mark: 
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

- [REQUIRED] You're required to render 5 most recent comments for post. This feature is yet to be implemented. Kindly attend to it. 

### Area of attention :point_down: 

comment


-  It should look something similar to this :point_down: 

```
   <% post.recent_comments.each do |comment| %>
       <p><%= comment.author.name %>: <%= comment.text %></p> 
   <% end %>

```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @BenM4k 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Changes requested have been implemented  ✅ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**